cognee/tasks/translation/translate_content.py

```
@@ -0,0 +1,385 @@
+    lang_code, conf = detection
+    detected_language = _normalize_lang_code(lang_code)
+    try:
+        conf = float(conf)
+    except (TypeError, ValueError):
+        conf = 0.0
+    if math.isnan(conf):
+        conf = 0.0
+    conf = max(0.0, min(1.0, conf))
+    return detected_language, conf
+
+def _requires_translation(detected_language: str, target_language: str, conf: float, threshold: float, provider_name: str, text: str) -> bool:
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)